mlkem/indcpa.c

```diff
@@ -352,6 +352,26 @@ void gen_matrix(polyvec *a, const uint8_t seed[MLKEM_SYMBYTES],
 #endif /* MLKEM_USE_NATIVE_NTT_CUSTOM_ORDER */
 }
 
+// clang-format off
+STATIC_TESTABLE
+void matvec_mul(polyvec *out, polyvec const *a, polyvec const *v,
+    polyvec_mulcache const *vc)
```

Comment:
1) change to `const polyvec` rather than `polyvec const`?

2) Can this be a `const polyvec a[MLKEM_K]`?

Reply:
Please add a commen documenting the bounds.

Reply:
1. OK 2. OK 3. done